client/src/Components/SubmitButton.tsx

```diff
@@ -0,0 +1,11 @@
+export default function SubmitButton(){
+    const handleSubmit = () => {
+    //to be added: send cart items to db
```

Comment:
is this a to do for another issue?

Reply:
yes. @Leewiz  is continuing with this one. thanks!